cpp/src/parquet/file_reader.h

```diff
@@ -58,6 +62,10 @@ class PARQUET_EXPORT RowGroupReader {
   // column. Ownership is shared with the RowGroupReader.
   std::shared_ptr<ColumnReader> Column(int i);
 
+  // Construct a RecordReader for the indicated row group-relative column i.
+  // Ownership is shared with the RowGroupReader.
```

Comment:
```suggestion
  // EXPERIMENTAL: Construct a RecordReader for the indicated column of the row group.
  // Ownership is shared with the RowGroupReader.
```

Reply:
Or it would be good to explicitly let user know the return type is an internal class.

Reply:
Done.